src/Helpers/Helpers.tsx

```diff
@@ -29,6 +29,26 @@ export const tasksExecuteSucessNotif = (store, title, ids, task_id) => {
   });
 };
 
+export const remediationsCreatedNotif = (store, title, remediation_id) => {
```

Comment:
We could slightly improve the code if we would made some base function (similarly to `displayErrorNotification`) and then just reuse it here with specific parameters :)

Reply:
I don't think that it is necessary, these notifications are kinda special, and for normal notifications we can use `dispatchNotification` directly. However, there will be only 2 of those, so no generic function is needed IMO.

Reply:
Alright, makes sense